src/components/stepforms/convert-filter-step-tree.ts

```diff
@@ -53,6 +92,32 @@ export function buildConditionsEditorTree(
   }
 }
 
+/**
+buildConditionsEditorTree transform this:
+{
+	operator: 'and',
+	conditions: [{ column: 'toto', value: 'A', operator: 'eq' }],
+	groups: [
+		operator: 'or',
+		conditions: [
+      { column: 'tata', value: 'B', operator: 'ne' },
+      { column: 'titi', value: '2', operator: 'le' }
+    ],
+	]
+}
```

Comment:
There is always the `groups` key

```suggestion
{
	operator: 'and',
	conditions: [{ column: 'toto', value: 'A', operator: 'eq' }],
	groups: [
		operator: 'or',
		conditions: [
      { column: 'tata', value: 'B', operator: 'ne' },
      { column: 'titi', value: '2', operator: 'le' }
    ],
    groups: [],
	]
}
```

Reply:
Thanks ! I corrected it